### PR TITLE
Add Context.ai Integration

### PR DESCRIPTION
**Pull request by BarberAlec**
Context.ai is an evaluations and analytics tool for products powered by LLMs.

This integration is with Haystack 2.0.

### CODE REVIEW

File: integrations/context-ai.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - name: Henry Scott-Green
+      socials:
+        linkedin: https://www.linkedin.com/in/hcscottgreen/
+    - name: Amishapriya Singh
+      socials:
+        github: amisha29sh
+        linkedin: https://www.linkedin.com/in/amisha29/
+pypi: https://pypi.org/project/context-haystack/
+repo: https://github.com/contextco/context-haystack
+type: Analytics Framework
```
Comment (TuanaCelik) [inline integrations/context-ai.md:23]: Heya, we have 2 tags that might work for you, can you please select the one that makes most sense: 
"Monitoring Tool" or "Evaluation Framework"

File: integrations/context-ai.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+response = pipe.run(
+    data={
+        "prompt_builder": {"template_variables":{"location": location}, "prompt_source": messages},
+        "prompt_analytics": context_parameters,
+        "assistant_analytics": context_parameters,
+    }
+)
+
+print(response)
+```
```
Comment (TuanaCelik) [inline integrations/context-ai.md:102]: If it makes sense, could you add a section on licensing here at the end to tell people what license this is distributed under? Example" https://haystack.deepset.ai/integrations/chroma-documentstore

File: integrations/context-ai.md (hunk 1 above)
Comment (BarberAlec) [reply integrations/context-ai.md]: Hi Tuana! Thanks, I have updated the type field to "Monitoring Tool".

File: integrations/context-ai.md (hunk 2 above)
Comment (BarberAlec) [reply integrations/context-ai.md:103]: Thanks! I have added a license field here!

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**